CHANGELOG.rst

```diff
@@ -31,6 +31,7 @@ Fixed
 -----
 - all temporal model files are now deleted after stopping the Rasa server
 - ``rasa shell nlu`` now outputs unicode characters instead of ``\uxxxx`` codes
+- ``x in AnySlotDicw`` is now True for any x
```

Comment:
Can we come up with sth more self explanatory? 
Maybe sth like `removed empty slot warnings during interactive learning`

Reply:
makes sense, i added that